conversiontracerbullet/src/test/java/org/stanford/AuthDBPropertiesTest.java

```diff
@@ -28,23 +29,24 @@
 
     private AuthDBProperties authProps;
     private Properties serverConf;
-    private String propertyFile;
+    private String serverConfResourceName = "/server.conf";
     private File serverConfFile;
     private File tmpDir;
 
     @Before
     public void setUp() throws IOException, InvocationTargetException, IllegalAccessException, NoSuchMethodException, NoSuchFieldException {
+        tmpDir = createTempDirectory("AuthDBPropertiesTest_").toFile();
         authProps = new AuthDBProperties();
-        // Use private method to get the server.conf file path
-        Field f = AuthDBProperties.class.getDeclaredField("propertyFile");
-        f.setAccessible(true);
-        propertyFile = (String) f.get(authProps);
-        serverConfFile = new File(propertyFile);
-        // Use private method to get the server.conf properties
-        Method m = AuthDBProperties.class.getDeclaredMethod("loadDataSourceProperties", String.class);
+
+        Method m = AuthDBProperties.class.getDeclaredMethod("loadPropertyResource");
```

Comment:
If the data in the properties file changes without changing any hard-coded values in the test, the test breaks.  There are various ways to do this, for sure, and this one suffices to get the job done.  If using fixed values is required to merge this, we can do that, but it just shifts that data into two places (the resource file and the test, both of which only exist in the tests).